File: Assignment 2/if else/addition.cpp
#include<iostream>
using namespace std;
class Addition
{
    int num1,num2,result;
    public: void add()
    {
        cout <<"Enter any two numbers:\n";
        cin>>num1>>num2;
        result=num1+num2;
        cout<<"result:"<<result<<endl;
        if(result>100)
        {
            cout<<"The result is greater than 100";
        }
        else
        {
            cout<<"The result is not greater than 100";
        }
    }

};
int main()
{
    Addition obj;
    obj.add();
}

File: Java assignment(1-2)/java Assignment 2/Addition.java
import java.util.Scanner;
class Addition
{
 float n1,n2,add;
void accept()
{
Scanner sc=new Scanner(System.in);
System.out.println("Enter the two numbers to be added: ");
n1=sc.nextFloat();
n2=sc.nextFloat();
}
void display()
{
add=n1+n2;
System.out.println("The Addition of the two numbers is: "+add);
if(add>100)
{
System.out.println("The result of the addition is greater than 100");
}
else
{
System.out.println("The result is less than 100");
}
}
public static void main(String...args)
{
Addition obj=new Addition();
obj.accept();
obj.display();
}
}